lib/modulesync/git.rb

```diff
@@ -27,9 +28,25 @@ def self.pull(git_base, name, branch, opts)
       end
     end
 
+    def self.bump(repo, m)
+      m.bump!
+      puts "Bumped to version #{m.version}"
```

Comment:
This is reporting the old version instead of the new version, which is confusing when looking at the output.

It looks like puppet-blacksmith returns the new version from `bump!`, could we instead save new_version as a variable and report that instead of m.version?